### PR TITLE
Use ks-tool image to replace the personal image path

### PR DESCRIPTION
**Pull request by LinuxSuRen**

### CODE REVIEW

File: charts/ks-devops/templates/config-migrate-job.yaml [hunk 1]
```diff
@@ -8,8 +8,8 @@ spec:
     spec:
       containers:
         - name: migrate
-          image: ghcr.io/linuxsuren/ks/ks:v0.0.37
-          command: ["/ks", "option", "migrate", "--service", '{{ include "ks-devops.fullname" . }}-apiserver.{{ .Release.Namespace }}:9090', "--namespace", "{{ .Release.Namespace }}"]
+          image: "{{ .Values.image.registry }}/{{ tpl .Values.image.tools.repository . }}:{{ tpl .Values.image.tools.tag . }}"
```
Comment (JohnNiang) [inline charts/ks-devops/templates/config-migrate-job.yaml:11]: Why do we need the directive `tpl` here?

Comment (LinuxSuRen) [reply charts/ks-devops/templates/config-migrate-job.yaml:11]: This should be a mistake.

**Comment by JohnNiang** [comment]
Please remove the period `.` of the tail as well.

**Comment by JohnNiang** [comment]
LGTM

**Pull request merged**